Buildings/Fluid/HeatExchangers/Examples/WetCoilCounterFlowPControlAutoTuning.mo

```diff
@@ -1,6 +1,6 @@
 within Buildings.Fluid.HeatExchangers.Examples;
 model WetCoilCounterFlowPControlAutoTuning
-  "Model that demonstrates use of a heat exchanger with condensation and with autotuning feedback control"
+  "Model that demonstrates the use of a heat exchanger with condensation and autotuning feedback control"
```

Comment:
`and "with" autotuning feedback control`

Reply:
Add "with"